### ISSUE TITLE
Have to re run installer to to get xlights to work.

### ISSUE THREAD

**Issue by myfyn69** [smeighan/xLights#2282]
When opening x-lights everything opens but if you click on anything it freezes. to solve this I have to rerun the installer. and usually it fixes the problem. also for no apparent reason when clicking the faces option it also freezes and goes to a not responding message. and has to be forced closed.is there a log file or crash report file that I can add to the report somehow?

**Comment by keithsw1111** [comment]
Tools ... package logs.

**Comment by myfyn69** [comment]
[xLightsProblem.zip](https://github.com/smeighan/xLights/files/5361634/xLightsProblem.zip)
 Please let me know if this worked!

**Comment by keithsw1111** [comment]
The logs are good but they dont help. There is only one execution here and they only go back 20 mins. I really need you to capture the logs just after the hang.
logs are in c:\users\<you>\appdata\roaming\xlights_log4cpp.log

**Comment by myfyn69** [comment]
Okay eating lunch and then I’ll work on a song and see if I can replicate the issue.

**Comment by myfyn69** [comment]
I attached a picture  to show i can find the files but when i try to add them here it says its not a file that is supported.

[xLights_l4cpp.log](https://github.com/smeighan/xLights/files/5361847/xLights_l4cpp.log)
![Roaming 10_11_2020 12_43_49 PM](https://user-images.githubusercontent.com/72671845/95688532-e004c480-0bbf-11eb-8620-946f2a6d6030.png)

**Comment by myfyn69** [comment]
Just after typing the above i went back into xlights that has been open and clicked on sequence and it froze. this is the log file in the roaming that it made.

[xLights_l4cpp.log](https://github.com/smeighan/xLights/files/5361867/xLights_l4cpp.log)

**Comment by myfyn69** [comment]
[xLightsProblem.zip](https://github.com/smeighan/xLights/files/5361868/xLightsProblem.zip) this is from doing the package logs just after reopening from the crash.

**Comment by keithsw1111** [comment]
Its hard to tell ... it would be useful if you has a timeline of what you did as the logs seem to suggest you did a bunch of stuff that you dont talk about above. Most of that is likely irrelevant but finding root cause requires that actions are kept super simple from xLights start to hang.
That being said from what i can see it looks to me like it may be video card driver related. I am not super confident about that but the last line seems to occur just prior to some video card activity. Have you tried updating your video card drivers to the latest from your card vendor ... not those from windows update.

**Comment by myfyn69** [comment]
Yes the driver is up to date. I had to update it to run my flight sim and that is also the computer I have x-lights on. xlights has been reopened since the crash as I am trying to remember how to use a remote pi run as a matrix and not just play the mp4s from the sequences. that is a different topic that is not for here. just before the last crash I was using fpp connect to upload my setup to our 2 pi's then clicked on the sequencer tab.

**Comment by myfyn69** [comment]
[xLights_l4cpp.log](https://github.com/smeighan/xLights/files/5362207/xLights_l4cpp.log)
 go ahead and check this one I have been able to recreate it a few times now. When I open x-lights everything starts up fine. then I clicked on sequence and sometimes it hangs but the last 3 times it was fine. I opened a sequence. rendered it then played it and everything worked. I then undocked the house preview and moved it to a different screen and that's when it froze. I actually Tired this a few times. I am able to keep it on my main screen and even undock the house preview. but as soon as i move either to a different screen it hangs and is now ctd. also I double checked my drivers and a new one came out on the 7th so I installed that one. also I dont think it will help but ill explain my comp setup to see it that helps at all. I have a brand new Alienware aroura desktop with a gtx 2080 super. it is running 4 screens. one is a 42" tv as the main monitor, and then I have two 32" tv's as 2nd and 3rd monitors and then a 4th 7" screen that I use for my gps in my plane sim. I don't think that should matter but I'm not a computer guy so not sure.

**Comment by keithsw1111** [comment]
The way you describe the problem it definitely sounds like a driver issue. Maybe there are still bugs in the current version of the drivers.
